src/crepe/api/Asset.cpp

```diff
@@ -0,0 +1,58 @@
+#include <filesystem>
+#include <stdexcept>
+#include <whereami.h>
+
+#include "Asset.h"
+#include "api/Config.h"
+
+using namespace crepe;
+using namespace std;
+
+Asset::Asset(const string & src) : src(find_asset(src)) { }
+Asset::Asset(const char * src) : src(find_asset(src)) { }
```

Comment:
Als ik echt heel erg mag muggenziften, dan hoort er nog een enter tussen deze twee constructors.

Reply:
Het staat nergens in de code standaard en ze zijn identiek (op type na) dus ik laat ze liever zo staan.